File: src/main/java/pl/training/workshop/m6/s04_encapsulatefactory/SeatSale.java
package pl.training.workshop.m6.s04_encapsulatefactory;

import java.util.List;

import pl.training.workshop.shared.Money;

/** Stabilny kontrakt sceny: sprzedaż miejsc na jeden seans (rząd 10+ to VIP). */
public record SeatSale(String title, Money base, List<Integer> rows) {
}

File: csharp/src/Training.Workshop/M6/S04EncapsulateFactory/SeatSale.cs
using Training.Workshop.Shared;

namespace Training.Workshop.M6.S04EncapsulateFactory;

/// <summary>Stabilny kontrakt sceny: sprzedaż miejsc na jeden seans (rząd 10+ to VIP).</summary>
public sealed record SeatSale(string Title, Money Base, IReadOnlyList<int> Rows);
